tests/test_telemetry.py

```diff
@@ -21,13 +21,21 @@ def validator(data):
         interfaces.library.add_telemetry_validation(validator, is_success_on_expiry=True)
         interfaces.agent.add_telemetry_validation(validator, is_success_on_expiry=True)
 
-    @missing_feature(library="dotnet")
-    @missing_feature(library="python")
     def test_telemetry_proxy_enrichment(self):
         """Test telemetry proxy adds necessary information"""
         interfaces.agent.assert_headers_presence(
-            path_filter="/api/v2/apmtelemetry",
-            request_headers=["dd-agent-hostname", "dd-agent-env", "datadog-container-id"],
+            path_filter="/api/v2/apmtelemetry", request_headers=["dd-agent-hostname", "dd-agent-env"],
+        )
+        interfaces.agent.assert_headers_match(
+            path_filter="/api/v2/apmtelemetry", request_headers={"via": r"trace-agent 7\..+"},
```

Comment:
@cbeauchesne this test will fail on the current version of agent 7.36.x. 
IIRC there was an annotation, to mark failing releases. I'll look it up.

It is working correctly in version 7.35.x and below and it should be fixed in the version 7.37

Reply:
the good annotation is something like @bug(context.agent >= "7.36.0" and  context.agent < "7.37.0")